core/src/models/AppRefreshQuery.ts

```diff
@@ -90,7 +90,12 @@ export class AppRefreshQuery extends LoggedModel<AppRefreshQuery> {
 
   @BeforeSave
   static async noUpdateIfLocked(instance: AppRefreshQuery) {
-    await LockableHelper.beforeSave(instance, ["state"]);
+    await LockableHelper.beforeSave(instance, [
+      "state",
+      "value",
+      "lastChangedAt",
+      "lastConfirmedAt",
+    ]);
```

Comment:
It's a little odd that there wasn't a test that tried to do an update and failed before